test/actor_test.py

```diff
@@ -1290,6 +1296,19 @@ def inc(self):
     # Submit some new actor tasks.
     x_ids = [actor.inc.remote() for _ in range(5)]
 
+    # Make sure that getting the result raises an exception.
+    for _ in range(10):
```

Comment:
why is this loop needed?

Reply:
Not sure, I was just imitating what was already here, I guess. I think it may just be checking that the exception gets re-raised if you call `ray.get` again.

Reply:
Hm. I added the loop in the test below. It may not be necessary, but I think it's possible that some calls to `ray.get` could go to the raylet before the plasma store disconnects and some (most) calls would go to the raylet after the store disconnects, so they might test slightly different things. However, the loop may not add any value.

Reply:
Could that case (`ray.get` reaching raylet before plasma store disconnects) be avoided if we use `cluster_utils` and `remove_node` for this test?

I'm seeing a weird issue with this case in #3322. But when I change this test to use cluster_utils, the issue is gone. I didn't figure out why. See https://github.com/ray-project/ray/pull/3332#issuecomment-440300929, and https://github.com/ray-project/ray/pull/3332#issuecomment-440514232.

Reply:
Hmm how about I change this so that `x_ids = [ ... ]` goes inside the loop? It would be nice if we could test submitting methods before and after the node is detected to be dead (this is why I added the repeated block of code here).

Reply:
I am in favor of using `cluster_utils` and generally deprecating `ray._init(num_local_schedulers=...)`. However, I don't know why that would affect the test.